SECURITY.md

```diff
@@ -0,0 +1,8 @@
+# How to report a security issue with authd-msentraid
```

Comment:
This is generic and not only the msentra id brokers. Do you mind rephrasing that this is for any brokers here?

Or this line could be with "authd oidc-based broker" and then, we can have one line listing them (authd-msentraid for now).

Reply:
It is targetting the [msentraid](https://github.com/ubuntu/authd-oidc-brokers/tree/msentraid) branch and matching the name set in the snapcraft.yaml but I'm fine changing it to be generic if you prefer

Reply:
Right, I don’t think people will really go to each branch to read about security.md for that specific branch, also GitHub integrates when you open an issue a read of `security.md` which seems to be following the content in HEAD (like it does for the bug template). So I would:
1. Target only the main branch (automerge will copy them in dedicated branches anyway)
2. Have generic content there, that will cover all brokers